### ISSUE TITLE
Pi-hole 5.2 Support

### ISSUE THREAD

**Issue by vmstan** [vmstan/gravity-sync#116]
https://pi-hole.net/2020/11/28/pi-hole-core-web-v5-2-and-ftl-v5-3-released/

I have not validated Gravity Sync support yet, as I'm waiting on the official Docker image to be updated. However, I do not anticipate any issues. 

If you do encounter any please comment below.

**Comment by vmstan** [comment]
Known issue: the new CNAME DNS records feature is not replicated by Gravity Sync. The data is stored in `/etc/dnsmasq.d/05-pihole-custom-cname.conf` which is not monitored.

**Comment by vmstan** [comment]
**FAQ:** Regarding mixed mode Pi-hole versions (upgrading your primary to 5.2 and leaving secondary at a previous version) this appears to work, but I would suggest temporarily disabling smart sync while in this mode, and only pulling from the upgraded version to the older.

**Comment by timothystewart6** [comment]
Looks like the docker image was updated.  Thank you for this!  Also, do you plan on supporting `CNAME`s?

**Comment by vmstan** [comment]
Yes, I will add CNAME support in the next release, but I need to have a rethink of that code to make it more flexible to handle adding more of these types of files. I've been wanting to add some replication structure to other files in the `/etc/dnsmasq.d` folder for a bit, so this just makes it more of an imperative.

**Comment by timothystewart6** [comment]
Nice.  Thank you.  When I read https://github.com/vmstan/gravity-sync/issues/82 I thought the same thing.  Thank you for this.  It's really nice to have as an option.

**Comment by michabbs** [comment]
I have my own custom `/etc/dnsmasq.d/99-local.conf` file and would like to sync. Please think about it. :-)
On the other hand I can imagine someone have a local config file on one PH and do not want to migrate it to the second PH. Please think about it too. :-)

**Comment by vmstan** [comment]
3.2.0 is live

**Comment by hansenc0705** [comment]
@vmstan is there another issue or a place that we could track support for CNAME Records? Thanks!

**Comment by vmstan** [comment]
It's there now as of 3.2.
https://github.com/vmstan/gravity-sync/wiki/Changelog

**Comment by hansenc0705** [comment]
my bad, thanks @vmstan.
